### ISSUE TITLE
UnknownApiNameOrVersion: name: earthengine  version: v1alpha

### ISSUE THREAD

**Issue by habibutsu** [google/earthengine-api#157]
After updating to latest version `google-api-python-client` (or installing in a new environment) the following error appeared. 

```python
UnknownApiNameOrVersion: name: earthengine  version: v1alpha
```
As I understand, it happens because [google-api-python-client](https://github.com/googleapis/google-api-python-client) since 2.0, for api discovering started to use static documents. Corresponded document `earthengine.v1alpha.json` is missed. Needed to do proposed recommendations for [migration](https://github.com/googleapis/google-api-python-client/blob/master/UPGRADING.md) to 2.0 or freeze version in requirements.

**Comment by giswqs** [comment]
I had the same issue. Downgrading `google-api-python-client` to 1.12.8 resolves the issue temporarily. 
https://github.com/giswqs/geemap/discussions/341

**Comment by simonff** [comment]
Yes, apologies for the breakage. This should be resolved next week.

**Comment by simonff** [comment]
The issue is fixed, please install the latest earthengine-api package 0.1.256.

**Comment by giswqs** [comment]
@simonff Thank you for the fix! I just tested v0.1.256, and I can confirm that the issue has been fixed.
